internal/logger/message/log/entry.go

```diff
@@ -22,11 +22,19 @@ import (
 	"time"
 )
 
+// ObjectVersion object version key/versionId
+type ObjectVersion struct {
+	ObjectName string `json:"objectName"`
+	VersionID  string `json:"VersionId,omitempty"`
```

Comment:
s/VersionId/versionId/

ObjectVersion is used in Args struct where fields does not start with a capital letter.

Reply:
@vadmeste can you send these changes, this PR is merged already.